docs/source/configuring_deploying.rst

```diff
@@ -307,3 +307,117 @@ setting has been updated to no longer include the old node's public key, the
+  INFO  | raft::raft:891       | [12797589408118497989] became leader at term 2
+
+On a node that becomes a follower and votes for another node, the logs will look
+like this:
+
+.. code-block:: console
+
+  INFO  | raft::raft:833       | [9142281103993713288] became follower at term 0
+  INFO  | raft::raft:433       | [9142281103993713288] newRaft [peers: [9142281103993713288, 12797589408118497989], term: 0, commit: 0, applied: 0, last_index: 0, last_term: 0]
+  INFO  | raft::raft:833       | [9142281103993713288] became follower at term 1
+  INFO  | raft::raft:1014      | [9142281103993713288] [term: 1] received a MsgRequestVote message with higher term from 12797589408118497989 [term: 2]
+  INFO  | raft::raft:833       | [9142281103993713288] became follower at term 2
+  INFO  | raft::raft:1181      | [9142281103993713288] [logterm: 1, index: 2, vote: 0] cast MsgRequestVote for 12797589408118497989 [logterm: 1, index: 2] at term 2
+
+At this point, the Raft engine is running and ready to handle blocks.
+
+
+Troubleshooting and Common Issues
+=================================
+
+Below is a list of common issues that are seen when attempting to start and run
+a Sawtooth Raft network. If you encounter any problems, please consult this list
+to see if your issue is discussed. If you are still having problems, find out
+how to get help in the :doc:`/community/join_the_discussion` section.
+
+
+No Leader Is Elected
+--------------------
+
+When a Sawtooth Raft network is unable to elect a leader, one or more nodes will
+continuously start new elections. This is usually a sign that the nodes are not
+peering properly. If you encounter this problem, see `Troubleshooting
+Connectivity`_ for tips on how to resolve it.
+
+
+Troubleshooting Connectivity
+----------------------------
+
+
+Validator Peering Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are experiencing network peering problems, the first thing to do is to
+verify that the validators are configured to peer properly. If the validators
+are not started up with static peering or the list of all existing validator
+endpoints is not supplied to a validator, some nodes will not be able to
+communicate. See the section on `Validator Peering Requirements`_ to ensure that
+you are providing the correct parameters for the validators to peer properly.
+
+
+Max Peers Exceeded
+~~~~~~~~~~~~~~~~~~
+
+If you are running a large Raft network (around 10 nodes), you might see an
+error in one or more of the validators' logs indicating that a peering request
+was rejected because the validator has reached its maximum number of peers. To
+resolve this issue, you will need to increase the peer limit for all validators.
+If you are starting the validator from the command line, you can specify the
+maximum number of peer connections with the ``--maximum_peer_connectivity``
+option (see the `sawtooth-validator documentation`_ for more details).
```

Comment:
I think we should also include a section on what happens when the `sawtooth.consensus.raft.peers` setting is incorrect and how to check it.